src/hooks/engine.rs

```diff
@@ -2376,6 +2398,7 @@ pub mod exported {
             let marker = MainThreadMarker::new();
 
             campath::override_view(marker);
+            tas_studio::tas_playback_rendered_views(marker);
```

Comment:
Should this be before or after campath?

Reply:
I cannot think of how this would matter. The only time both of these are called is when someone loads a campath and then play a campath while they also are playing back TAS views. Talking about stretch.